model/flow/identifierList.go

```diff
@@ -36,3 +36,12 @@ func (il IdentifierList) Less(i, j int) bool {
 func (il IdentifierList) Swap(i, j int) {
 	il[j], il[i] = il[i], il[j]
 }
+
+func (il IdentifierList) String() []string {
```

Comment:
`String()` typically returns a single `string`. Maybe call this `Strings`?

Reply:
Fixed in https://github.com/onflow/flow-go/pull/211/commits/74812379d5b4f6200630ea2184943a9ea04df029.